### DISCUSSION TITLE
API error on updating Development branch on Github Issue description page for any repo

### DISCUSSION THREAD

**Discussion by gauravparmar** [community/community#39908] [Projects and Issues]
### Select Topic Area

Bug

### Body

On clicking this gear icon on top right and choosing a different branch, the updated branch is not coming and the respective API throws error -
![image](https://user-images.githubusercontent.com/2599630/203717510-555d18ab-9436-4d54-8891-93faa67fc8ee.png)
![image](https://user-images.githubusercontent.com/2599630/203718023-66f32ecd-56ea-488e-ad5b-76e14f4b71da.png)

![image](https://user-images.githubusercontent.com/2599630/203717768-ddfcdf97-152d-413d-a2a1-967f01a668cd.png)

**Comment by ossanna16** [comment]
Hi there @gauravparmar and welcome to our community! Thank you for reporting this bug!
